pkg/api/server/server.go

```diff
@@ -147,8 +149,26 @@ func newServer(runtime *libpod.Runtime, duration time.Duration, listener *net.Li
 	return &server, nil
 }
 
-// Serve starts responding to HTTP requests
+// If the SDNOTIFY_SOCKET is set, communicate the PID and readiness, and
+// further unset the SDNOTIFY_SOCKET to prevent containers from sending
```

Comment:
I'm still struggling to understand the entire ramifications of this PR, but while I do so, you may want to remove the misleading `SD` from the envariable name in the comments above.